app/screens/MyLocation/__tests__/MyLocationScreen.tests.tsx

```diff
@@ -10,3 +12,27 @@ it("MyLocationScreen renders correctly", () => {
   const tree = create(<MyLocationScreen />).toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+it("Change current location to ireland", () => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
`import * as redux from 'react-redux'`

Then `const useSelectorMock = jest.spyOn(redux, "useSelector");` so you don't need the `// eslint-disable-next-line @typescript-eslint/no-var-requires`